### PR TITLE
import into: add known issue

### PR DESCRIPTION
**Pull request by D3Hunter**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-import-into.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 > **警告：**
 >
 > 目前该语句为实验特性，不建议在生产环境中使用。
+>
+> 已知问题：任务启动后，对要导入的数据在本地进行排序时，如果使用的磁盘空间超过了设定的 disk-quota 值或本地磁盘空间的 80% ，并且开始向 TiKV 写入数据时，如果取消该任务或该任务运行失败，后台 routine 将会继续运行一段时间，然后才会真正退出。参考 [#45048](https://github.com/pingcap/tidb/issues/45048)。
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:14]:
```suggestion
> 已知问题：导入任务启动后，在对要导入的数据进行本地排序期间，当使用的 TiDB 磁盘空间超过了设定的 disk-quota 值或本地磁盘空间的 80% 并且 TiDB 已经开始向 TiKV 写入数据时，如果取消该任务或导入任务运行失败，后台 routine 将会继续运行一段时间，然后才会真正退出。参考 [#45048](https://github.com/pingcap/tidb/issues/45048)。
```

File: sql-statements/sql-statement-import-into.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 > **警告：**
 >
 > 目前该语句为实验特性，不建议在生产环境中使用。
+>
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:13]: 这里 “设定的 disk-quota 值“ 是指 TiDB Lightning 的[磁盘资源配](https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-physical-import-mode-usage#%E7%A3%81%E7%9B%98%E8%B5%84%E6%BA%90%E9%85%8D%E9%A2%9D-%E4%BB%8E-v620-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5)额吗？

Comment (D3Hunter) [reply sql-statements/sql-statement-import-into.md:13]: 1 是 import into 的配额
影响就是，用户虽然看到看到任务在 cancel 或失败状态了，但导入过程还会继续一段时间

后台 routine，即后台线程

Comment (D3Hunter) [reply sql-statements/sql-statement-import-into.md:13]: 可以

Comment (qiancai) [reply sql-statements/sql-statement-import-into.md:13]: 是通过哪个配置项设定呢

**Review by Frank945946 — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
/lgtm

File: sql-statements/sql-statement-import-into.md [hunk 3]
```diff
@@ -20,6 +20,10 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 
 - 对于存储在 TiDB 本地的数据文件，`IMPORT INTO` 仅支持在当前用户连接的 TiDB 节点上运行，因此数据文件需要存放在当前用户连接的 TiDB 节点上。如果是通过 PROXY 或者 Load Balancer 访问 TiDB，则无法导入存储在 TiDB 本地的数据文件。
 
+## 已知问题
+
+数据导入任务启动后，在对要导入的数据进行本地排序期间，当使用的 TiDB 磁盘空间超过了 [`DISK_QUOTA`](#withoptions) 的设定值或本地磁盘空间的 80%，并且 TiDB 已经开始向 TiKV 写入数据时，如果取消该任务或导入任务运行失败，后台导入线程将会继续运行一段时间，然后才会真正退出。详情参考 [#45048](https://github.com/pingcap/tidb/issues/45048)。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-import-into.md:25]:
```suggestion
数据导入任务启动后，在对要导入的数据进行本地排序期间，当使用的 TiDB 磁盘空间超过了 [`DISK_QUOTA`](#withoptions) 的设定值或本地磁盘空间的 80%，并且 TiDB 已经开始向 TiKV 写入数据时，如果取消该任务或导入任务运行失败，后台导入线程会继续运行一段时间，然后才会真正退出。详情参考 [#45048](https://github.com/pingcap/tidb/issues/45048)。
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**